charts/product-console/templates/configmap.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "product-console.fullname" . }}
+  labels:
+    {{- include "product-console.labels" . | nindent 4 }}
+data:
+  {{- range $key, $value := .Values.configmap }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)